staging/src/k8s.io/dynamic-resource-allocation/structured/allocator.go

```diff
@@ -1315,3 +1326,21 @@ func containsNodeSelectorRequirement(requirements []v1.NodeSelectorRequirement,
 	}
 	return false
 }
+
+type bitfield int32
```

Comment:
```suggestion
type bitfield uint32
```

Reply:
It would probably make sense to test if this enough. We might need to restrict max number of claims (per pod/request?) somewhere if it's not done yet.